### DISCUSSION TITLE
How to apply the lower bound constraint in a ratio?

### DISCUSSION THREAD

**Discussion by Mithil15** [opencobra/cobrapy#1043] [Q&A] [answered]
Hello everyone,
Is it possible to constraint the lower bound of two reactions in a ratio?

for example, the following is my script:
```
for i in np.arange(start = 2.5, stop = 3.5, step = 0.5):
         model.reactions.<react_1>.lower_bound/model.reactions.<react_2>.lower_bound = i/i*3

```
However, it shows me the error:
`SyntaxError: cannot assign to operator`

Is there any other way to carryout this operation?

It would be helpful for me, if I get any direction to overcome the problem.

Thank you in advance

**Comment by matthiaskoenig** [comment, marked as answer]
I am not completely sure what you want to achieve.
`i/i*3` is always 3 for every loop iteration ?!

You can easily set one lower bound based on another bound.
```
r1.lb/r2.lb = value
```
could be written as
```
r1.lb = r2.lb * value
```
Then you would just do something like:
```
r1.lower_bound = -100
for i in np.arange(start = 2.5, stop = 3.5, step = 0.5):
    value = i  # here you would calculate your fraction
    r2.lower_bound = r1.lower_bound * i;
```
Not sure this is what you wanted to do.

**Reply by Mithil15** [reply]
Thank You Very much for the Answer, this is what I was asking for

**Reply by Midnighter** [reply]
@Mithil15 you can mark the above answer as solving your question.
